pkg/apis/scheduling/types.go

```diff
@@ -111,6 +114,12 @@ const (
 
 	// NotEnoughPodsReason is probed if there're not enough tasks compared to `spec.minMember`
 	NotEnoughPodsReason string = "NotEnoughTasks"
+
+	// NotEnoughPodsOfTaskReason is probed if there're not enough pods of task compared to `spec.minTaskMember`
+	NotEnoughPodsOfTaskReason string = "NotEnoughPodsOfTask"
```

Comment:
what does this mean?

Reply:
"NotEnoughPodsOfTask" exist in pkg/apis/scheduling/v1beta1/types.go, sync it to here.